src/addedComponents/EntityAssertionsWidget/usePluginComponent.ts

```diff
@@ -0,0 +1,14 @@
+import { usePluginComponent as realUsePluginComponent } from '@grafana/runtime';
+
+export function usePluginComponent<T extends object>(
+  id: string
+): {
+  isLoading: boolean;
+  component: React.ComponentType<T> | null | undefined;
+} {
+  if (realUsePluginComponent) {
+    return realUsePluginComponent<T>(id);
+  } else {
```

Comment:
Why do we need to wrap it like this? I what case `usePluginComponent` is not defined?

Reply:
Yeah we don't need that in fairness - API has been pushed about 9 months ago! I'll remove it.